pkg/tsdb/influxdb/response_parser_test.go

```diff
@@ -91,6 +91,82 @@ func TestInfluxdbResponseParser(t *testing.T) {
 		}
 	})
 
+	t.Run("Influxdb response parser should parse metricFindQueries normally", func(t *testing.T) {
+		parser := &ResponseParser{}
```

Comment:
nit: I can see this test covers two different use cases. Can we wrap them under two different `t.Run` so that next time we know what we are actually testing.

Reply:
Done.